extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+/**
+ * Gets Optimizely ID for Toolbar A/B test
+ * @returns {string}
+ */
+ve.init.wikia.getToolbarABTestId = function () {
+	return window.wgDevelEnvironment ? '4721410313' : '4701112678';
+};
+
+/**
+ * @returns {number} The A/B test variant number for the Toolbar A/B test
+ */
+ve.init.wikia.getToolbarABTestVariantNumber = function () {
+	var optimizely = window.optimizely,
+		optimizelyId = ve.init.wikia.getToolbarABTestId();
+
+	if ( optimizely && optimizely.variationMap && optimizely.variationMap.hasOwnProperty(optimizelyId) ) {
+		return optimizely.variationMap[optimizelyId] || 0;
+	}
+
+	return 0;
+};
+
+/**
+ * Hook for activating experiment on VE load
+ */
+mw.hook( 've.activate' ).add( function () {
+	window.optimizely.push( ['activate', ve.init.wikia.getToolbarABTestId()] );
+});
```

Comment:
I'm sorry that maybe linter is not catching it up for you automatically right now - I will look into it soon - perhaps this week.
If you want to run linter locally just take a look at ```.travis.yml``` and follow the same steps.